Azure/azure-cli
# [SQL] Add federated client id support for sql

**Description**
This PR adds server level support for federated client id used in cross tenant CMK scenario

**Testing Guide**
az sql server create --federated-client-id should create a server with corresponding federated client id value
az sql server update--federated-client-id should update a server with new federated client id value

**History Notes**
[SQL] `az sql server create/update`: Add federated client id support.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
